### PR TITLE
[DEV-1336] Fix change password page and changed password page

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes

Fix change password page and changed password page
This PR depends on #601, #604 , #605 

#### Screenshots (if appropriate):
image

image

image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tommaso1 — APPROVED** [review]

File: apps/nextjs-website/src/config.ts [hunk 1]
```diff
@@ -41,6 +41,27 @@ export const defaultOgTagImage = `${baseUrl}/images/dev-portal-home.jpg`;
...
+  'product_updates',
+  'api_keys',
+  'support',
+];
+
+export const webinarQuestionConfig = {
+  url: process.env.NEXT_PUBLIC_WEBINAR_QUESTION_URL,
+  resource: process.env.NEXT_PUBLIC_WEBINAR_QUESTION_SHEET_NAME,
+};
+
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/config.ts:64]:
```suggestion
```

**Review by MarcoPonchia — APPROVED** [review]

File: apps/nextjs-website/src/messages/it.json [hunk 2]
```diff
@@ -226,21 +226,21 @@
...
-            "checkEmailStart": "Se esiste un account associato a ",
             "checkEmailEnd": "riceverai una email con un link per impostare una nuova password.",
-            "resendEmailPrompt": "Non hai ricevuto l'email? Controlla nella posta indesiderata oppure",
-            "resendEmail": "Reinvia email",
-            "wrongEmail": "L'indirizzo email è errato?",
-            "passwordSet": "Password impostata correttamente",
+            "checkEmailStart": "Se esiste un account associato a ",
+            "checkEmailTitle": "Controlla l'email",
+            "emailFieldError": "Inserisci un indirizzo email valido",
+            "goToLogin": "Vai al login",
```
Comment (kin0992) [inline apps/nextjs-website/src/messages/it.json:234]:
```suggestion
            "goToLogin": "Torna al login",
```
on Figma there is this copy

File: apps/nextjs-website/src/messages/it.json [hunk 3]
```diff
@@ -226,21 +226,21 @@
...
-            "checkEmailTitle": "Controlla l'email",
-            "checkEmailStart": "Se esiste un account associato a ",
             "checkEmailEnd": "riceverai una email con un link per impostare una nuova password.",
-            "resendEmailPrompt": "Non hai ricevuto l'email? Controlla nella posta indesiderata oppure",
-            "resendEmail": "Reinvia email",
-            "wrongEmail": "L'indirizzo email è errato?",
-            "passwordSet": "Password impostata correttamente",
+            "checkEmailStart": "Se esiste un account associato a ",
+            "checkEmailTitle": "Controlla l'email",
+            "emailFieldError": "Inserisci un indirizzo email valido",
```
Comment (kin0992) [inline apps/nextjs-website/src/messages/it.json:233]:
```suggestion
            "emailFieldError": "L’indirizzo email non è valido",
```
on Figma there is this copy

**Comment by kin0992** [comment]
Take note that these steps can cause some issues (we need to investigate if it's true):
- insert the email address with a typo (e.g. mario instead of marco)
- the next screen shows me the email address inserted in bold and I see the error
- come back to the previous step, using the "Torna indietro" button
- I fix the email address -> in the query param there is the old email address, the one with the typo.
If that value is used then, we could have some issues

**Comment by marcobottaro** [comment]
> Take note that these steps can cause some issues (we need to investigate if it's true):
> 
> * insert the email address with a typo (e.g. mario instead of marco)
> * the next screen shows me the email address inserted in bold and I see the error
> * come back to the previous step, using the "Torna indietro" button
> * I fix the email address -> in the query param there is the old email address, the one with the typo.
>   If that value is used then, we could have some issues

The route replace was useless. I removed it, so the problem is solved. Thanks!

**Review by tommaso1 — APPROVED** [review]

**Pull request merged**